third_party/move/move-compiler-v2/src/file_format_generator/function_generator.rs

```diff
@@ -36,7 +40,10 @@ pub struct FunctionGenerator<'a> {
     /// A map from a temporary to information associated with it.
     temps: BTreeMap<TempIndex, TempInfo>,
     /// The value stack, represented by the temporaries which are located on it.
-    stack: Vec<TempIndex>,
+    /// Each temporary is paired with a bool, indicating whether the temporary is maybe stack-only
+    /// value (true) or definitely not a stack-only value (false).
```

Comment:
Would be good to say what a 'stack-only' value is, specifically in relation with 'definitely not'. Does false mean it is definitely in a local? Does true mean it maybe in a local or not?

Also we do already have `TempInfo` which tells the local associated with value, isn't this already telling you if its in a local or not? (TempInfo available ==> is in local; if thats not working as expected, perhaps need a fix?)